### PR TITLE
docs: Add Next.js route example to node/serverless

### PR DESCRIPTION
**Pull request by AryanJ-NYC**

### CODE REVIEW

File: src/collections/_documentation/platforms/node/serverless.md [hunk 1]
```diff
@@ -32,3 +32,31 @@ module.exports.hello = sentryHandler(async event => {
...
+
+## Vercel Next.js API Routes
+
+To set up Sentry error logging for a Next.js API route, build a wrapper (similar to the [AWS Lambda Sentry wrapper](#aws-lambda)).
+
+```javascript
+import Sentry from "@sentry/node"
+
+Sentry.init({
+  dsn: "YOUR DSN"
```
Comment (HazAT) [inline src/collections/_documentation/platforms/node/serverless.md:44]:
```suggestion
  dsn: "___PUBLIC_DSN___"
```

File: src/collections/_documentation/platforms/node/serverless.md [hunk 2]
```diff
@@ -32,3 +32,31 @@ module.exports.hello = sentryHandler(async event => {
   throw new Error("asd");
 });
 ```
+
+## Vercel Next.js API Routes
+
+To set up Sentry error logging for a Next.js API route, build a wrapper (similar to the [AWS Lambda Sentry wrapper](#aws-lambda)).
+
+```javascript
+import Sentry from "@sentry/node"
```
Comment (HazAT) [inline src/collections/_documentation/platforms/node/serverless.md:41]:
```suggestion
import * as Sentry from "@sentry/node"
```

File: src/collections/_documentation/platforms/node/serverless.md [hunk 3]
```diff
@@ -32,3 +32,31 @@ module.exports.hello = sentryHandler(async event => {
...
+      Sentry.captureException(error);
+      await Sentry.flush(2000);
+      throw error;
+    }
+  };
+}
+
+export default sentryHandler(async (req, res) => {
+  throw new Error("asd");
+});
```
Comment (HazAT) [inline src/collections/_documentation/platforms/node/serverless.md:61]: What is the purpose of this?

File: src/collections/_documentation/platforms/node/serverless.md [hunk 4]
```diff
@@ -32,3 +32,31 @@ module.exports.hello = sentryHandler(async event => {
...
+  dsn: "YOUR DSN"
+});
+
+function sentryHandler(routeHandler) {
+  return async (req, res) => {
+    try {
+      return await routeHandler(req, res);
+    } catch (error) {
+      Sentry.captureException(error);
+      await Sentry.flush(2000);
```
Comment (HazAT) [inline src/collections/_documentation/platforms/node/serverless.md:53]:
```suggestion
      // We wait up to 2000ms to flush the internal transport queue
      await Sentry.flush(2000);
```

File: src/collections/_documentation/platforms/node/serverless.md (hunk 3 above)
Comment (AryanJ-NYC) [reply src/collections/_documentation/platforms/node/serverless.md:61]: I wanted the documentation for NextJS to be as similar as possible to the [existing documentation for AWS Lambda](https://docs.sentry.io/platforms/node/serverless/).

**Pull request closed without merging**